### PR TITLE
User can search for new pokemon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
image

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -1 +1,15 @@
 /* exported data */
+var data = {
+  entries: [],
+  view: ''
+};
+
+window.addEventListener('beforeunload', function (event) {
+  var formJSON = JSON.stringify(data);
+  localStorage.setItem('javascript-local-storage', formJSON);
```
Comment (robgmerrill) [inline js/data.js]: Probably update this name from `'javascript-local-storage'` to `'party-local-storage'`

File: js/data.js [hunk 2]
```diff
@@ -1 +1,15 @@
...
+});
+
+var dataInJSON = localStorage.getItem('javascript-local-storage');
```
Comment (robgmerrill) [inline js/data.js]: Update `'javascript-local-storage'` to whatever name you use on line 9.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Local storage key name update for project suggestion.

**Comment by Nathan-Chon** [comment]
animation47

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**